contributing.md

```diff
@@ -0,0 +1,29 @@
+## Contributing
+
+We welcome contributions!
+
+We're using Github to write this guide as it's more likely to persist longer and allows for other people to discuss and contribute changes. It's a relatively new style of writing that most of us don't have experience with, so we're all experimenting on this together!
+
+A few tips on making good PRs:
```

Comment:
What do you think of this one? It's not too code-specific. https://github.com/blog/1943-how-to-write-the-perfect-pull-request